mysql/tls-bootstrap/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/spiffe/go-spiffe/v2/workloadapi"
+)
+
+const (
+	socketPath = "unix:///run/spire/sockets/agent.sock"
+	svidDir    = "/spire/certs"
+	bundleFile = "bundle.0.pem"
+	certFile   = "svid.0.pem"
+	keyFile    = "svid.0.key"
+)
+
+var (
+	certFilePath   = fmt.Sprintf("%s/%s", svidDir, certFile)
+	keyFilePath    = fmt.Sprintf("%s/%s", svidDir, keyFile)
+	bundleFilePath = fmt.Sprintf("%s/%s", svidDir, bundleFile)
+)
+
+func main() {
+	ctx := context.Background()
+	// Creates a new Workload API client, connecting to provided socket path
+	// Environment variable `SPIFFE_ENDPOINT_SOCKET` is used as default
+	client, err := workloadapi.New(ctx, workloadapi.WithAddr(socketPath))
+	if err != nil {
+		log.Fatalf("Unable to create workload API client: %v", err)
+	}
+	defer client.Close()
+
+	x509Context, err := client.FetchX509Context(ctx)
+	if err != nil {
+		log.Fatalf("Unable to fetch x509Context %v", err)
+	}
+
+	err = writeX509Context(x509Context)
+	if err != nil {
+		log.Fatalf("failed to write SVID/Bundle to disk: %v", err)
```

Comment:
```suggestion
		log.Fatalf("Failed to write SVID/Bundle to disk: %v", err)
```
nit: logs should be capitalized